docs/execution-modes.md

```diff
@@ -13,15 +13,15 @@ differences in underlying hardware (CPU, GPU, etc.), invalid data not present in
 your local 1%, rare corner cases, etc.
 
 But when it comes to building your code, correctness of your type hints and
-syntax, dependency packaging, Sematic can do a great deal to save you from a lot
+syntax, and dependency packaging, Sematic can do a great deal to save you from a lot
 of wasted time.
 
 Sematic provides the following execution stragegies:
 
 ### Silent execution
 
 Silent execution runs your code locally (i.e. on your machine) and does not
-write anything to the database. Your pipeline will not show up in the UI.
+write anything to the (local) database. Your pipeline will not show up in the UI.
```

Comment:
Nit: In this case, local or remote does not matter too much. We can leave this as is.

Reply:
Got it reverting it.